tests/src/Kernel/Framework/PersistedQueriesTest.php

```diff
@@ -115,4 +117,45 @@ public function persistedQueriesDataProvider(): array {
     ];
   }
 
+  /**
+   * Test the automatic persisted queries plugin.
+   */
+  public function testAutomaticPersistedQueries(): void {
```

Comment:
I think we should split this out into a `tests/src/Kernel/Framework/AutomatedPersistedQueriesTest.php`. That makes it easier to see what is part of APQ and what is part of traditional persisted queries.

To make sure we have sufficient test coverage we can look at the [apollo-server-integration-testsuit](https://github.com/apollographql/apollo-server/blob/main/packages/apollo-server-integration-testsuite/src/index.ts) and copy the tests related to APQ.